File: DivinoLanches/ClientApp/src/models/ProdutoComandaModel.ts
export class ProdutoComandaModel {

    id: number
    idComanda: number
    quantidade: number
    nomeProduto: string
    tipoProduto: string
    subTipoProduto: string
    valor: number

    constructor(obj: any) {
        this.id = obj.id
        this.idComanda = obj.idComanda
        this.quantidade = obj.quantidade
        this.nomeProduto = obj.nomeProduto
        this.tipoProduto = obj.tipoProduto
        this.subTipoProduto = obj.subTipoProduto
        this.valor = obj.valor
        return this
    }
}

File: DivinoLanches/Models/ProdutoComandaModel.cs
using System;
using System.Collections.Generic;

namespace DivinoLanches.Models
{
    public class ProdutoComandaModel
    {
        public int Id { get; set; }
        public int IdComanda { get; set; }
        public int Quantidade { get; set; }
        public string NomeProduto { get; set; }
        public string TipoProduto { get; set; }
        public string SubTipoProduto { get; set; }
        public double Valor { get; set; }
    }
}
